core/src/main/java/org/web3j/tx/Contract.java

```diff
@@ -430,4 +431,13 @@ public final String getDeployedAddress(String networkId) {
         return addr == null ? getStaticDeployedAddress(networkId) : addr;
     }
 
+    @SuppressWarnings("unchecked")
+    protected static <OriginalTypeT extends Type, NewTypeT> 
```

Comment:
Could we just use S & T here? I'm not so keen on lengthy parameterised type names.